taskcluster/ci/merge-mono/kind.yml

```diff
@@ -0,0 +1,118 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - translations_taskgraph.transforms.from_datasets:mono
+    - translations_taskgraph.transforms.find_upstreams:mono
+    - translations_taskgraph.transforms.command_context_from_params:transforms
+    - taskgraph.transforms.job:transforms
+    - translations_taskgraph.transforms.cache:transforms
+    - taskgraph.transforms.cached_tasks:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - clean-mono
+    - toolchain
+
+task-defaults:
+    label: merge-mono-{locale}
+    description: merge mono for {locale}
+    attributes:
+        stage: merge-mono
+        cache:
+            type: merge-mono
+            resources:
+                - pipeline/clean/merge-mono.sh
+
+    dataset-config:
+        substitution-fields:
+            - label
+            - description
+            - name
+            - treeherder.symbol
+            - worker.env
+            - upstreams-config.locale
+
+    upstreams-config:
```

Comment:
I _think_ you can forego the usage of this transform because your upstream is always the same. Instead, you should be able to directly add dependencies & fetches (similar to what we do for `train-vocab`, which is _always_ downstream of `merge-corpus`: https://github.com/mozilla/firefox-translations-training/blob/main/taskcluster/ci/train-vocab/kind.yml#L79-L89).